vendor/k8s.io/kubernetes/pkg/kubelet/cm/deviceplugin/manager_test.go

```diff
@@ -56,6 +56,7 @@ func TestNewManagerImplStart(t *testing.T) {
 // making sure that after registration, devices are correctly updated and if a re-registration
 // happens, we will NOT delete devices; and no orphaned devices left.
 func TestDevicePluginReRegistration(t *testing.T) {
+	t.SkipNow()
```

Comment:
Why are we not just taking https://github.com/kubernetes/kubernetes/pull/53028 which still runs the test (albiet with a potential delay)?

Reply:
Ah I was unware of that PR.  Lets just cherry pick that back instead.

Reply:
Seems we have better way forward that this PR, closing it.